src/client/app/components/pages/RootPage/index.tsx

```diff
@@ -29,7 +28,11 @@ import {
 } from '../../../util/types'
 import theme from '../../../theme'
 
-const Root = ({ store }) => (
+type RootProps = {
+  store: any
```

Comment:
is there a stricter subset of `any` that would work for root props? maybe the redux libraries export a store type that can be used here